### PR TITLE
ISPN-12630 SIFS does not set the seqId when loading index from disk

### PR DESCRIPTION
**Pull request by pruivo**
https://issues.redhat.com/browse/ISPN-12630

### CODE REVIEW

**Review by ryanemerson — APPROVED** [review]
LGTM, but I'll let @wburns take a look as he's spending time on the SIFS this week.

File: persistence/soft-index/src/main/java/org/infinispan/persistence/sifs/Index.java [hunk 1]
```diff
@@ -243,10 +255,10 @@ public void run() {
                      }
                      continue;
                   case STOP:
-                     assert indexQueue.poll() == null;
+                     assert indexQueue.peek() == null;
```
Comment (wburns) [inline persistence/soft-index/src/main/java/org/infinispan/persistence/sifs/Index.java:258]: :+1: Interesting little bug :D

**Review by wburns — COMMENTED** [review]
Looks good to me, just want to look over again before I integrate though.

**Comment by wburns** [comment]
Integrated into master, thanks @pruivo !

**Pull request merged**